scripts/analyse.sh

```diff
@@ -1,37 +1,93 @@
-#!/bin/sh
+#!/usr/bin/env bash
+# shellcheck disable=SC2016,SC2207,SC2002
 
-set -ex
+set -e
 
-BP=$(dirname $0)
+# generator_log_prefix="${1:-generator/tx-gen}"
+# explorer_log_prefix="${2:-node-on-explorer/node}"
+generator_log_prefix="${1:-logs/generator}"
+explorer_log_prefix="${2:-logs/node}"
 
-# extract txs from generator
-${BP}/xtx.sh generator/tx-gen-*.json > txs
-sort -k 1 -t ';' txs > txs1
+mkdir -p 'analysis'
+cd 'analysis'
+rm -f ./*
```

Comment:
this is kind of dangerous! imagine that the directly could not be created and cd into it. OK, we probably do not have access right then..

Reply:
if `cd` doesn't work, then `-e` above will terminate the script, and the `rm` will not happen -- so I think it's safe.